src/modules/displayScore.js

```diff
@@ -0,0 +1,24 @@
+const boardUser = document.querySelector('section .leader-board .board ul');
+
+export default class Displaying {
+  static special = false;
+
+  // eslint-disable-next-line class-methods-use-this
+  displayScore(user) {
+    const li = document.createElement('li');
+    if (Displaying.special) li.classList.add('special');
+    li.innerHTML = `${user.user} : ${user.score}`;
+    boardUser.appendChild(li);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  display(users) {
+    boardUser.innerHTML = '';
+    if (users.length > 0) {
+      users.forEach((user) => {
+        Displaying.special = !Displaying.special;
+        this.displayScore(user);
+      });
+    }
+  }
```

Comment:
- [ ] Kindly change all the functions in your code to the arrow function instead of the keyword function. This is part of the project requirements. Take a look please :point_down: 

![project-req](https://user-images.githubusercontent.com/26597432/202419490-f27aeaad-d457-4276-8a63-cdc6cc08c805.png)